.github/workflows/run-regression.yml

```diff
@@ -0,0 +1,35 @@
+name: Call Notification API QA Regression Tests
+
+on: 
+  workflow_call:
+    inputs:
+      environment:
+        description: 'Notification-api deployment environment to test (dev|perf|staging)'
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Notification-api deployment environment to test (dev|perf|staging)'
+        required: true
+        type: string
```

Comment:
```suggestion
```
Environment is a specific thing. [Not sure](https://docs.github.com/en/actions/using-jobs/using-environments-for-jobs) it should have type.

Reply:
again, in our case environment is just a string, and not really related to [deployment environment as intended by Github.](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment)

Reply:
type being changed to `environment`, [per the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputsinput_idtype).